RELEASE_NOTES.md

```diff
@@ -3,6 +3,28 @@ Release notes
 
 ### Next
 
+* Add support for the Address History user account creation attribute
+* Add ENVIRONMENT configuration option to replace hubSsoLocation and metadataUrl.
+* Send version number to hub
+* Support multitenancy
```

Comment:
Can we say "Support multiple services connected to GOV.UK Verify" like we do in the README?